console_libraries/menu.lib

```diff
@@ -61,6 +61,19 @@
 {{ template "_menuItem" (args . "cassandra.html" "Cassandra") }}
 {{ end }}
 
+{{ if query "up{job=~'blackbox.*'}" }}
```

Comment:
Right. I remember that. But I'm still not clear on what you want it to look like. I currently have the following config for my blackbox probers. Using my console configs I get three separate consoles - which is what I want. One shows my personal sites, one shows client sites (probed from a different location) and the latter shows the health of my isp's routers - a rather annoying necessity. Depending on what I'm debugging, I generally only want the info in one of those.

 ```
 - job_name: 'blackbox'
    metrics_path: /probe
    params:
      module: [http_2xx]
    target_groups:
      - targets:
        - web.site
        - another.site
        - etc.etc
    relabel_configs:
      - source_labels: [__address__]
        regex: (.*):80
        target_label: __param_target
        replacement: ${1}
      - source_labels: [__param_target]
        regex: (.*)
        target_label: instance
        replacement: ${1}
      - source_labels: []
        regex: .*
        target_label: __address__
        replacement: 127.0.0.1:9115

  - job_name: 'blackbox-ping'
    metrics_path: /probe
    params:
      module: [icmp]
    target_groups:
      - targets: ['hops.from', 'home.to', 'isp.egress']
    relabel_configs:
      - source_labels: [__address__]
        regex: (.*):80
        target_label: __param_target
        replacement: ${1}
      - source_labels: [__param_target]
        regex: (.*)
        target_label: instance
        replacement: ${1}
      - source_labels: []
        regex: .*
        target_label: __address__
        replacement: 127.0.0.1:9115

 - job_name: 'blackbox-client1'
    metrics_path: /probe
    params:
      module: [http_2xx]
    target_groups:
      - targets:
        - client1.host
        - client1.backup.host
        - client1.other.host
    relabel_configs:
      - source_labels: [__address__]
        regex: (.*):80
        target_label: __param_target
        replacement: ${1}
      - source_labels: [__param_target]
        regex: (.*)
        target_label: instance
        replacement: ${1}
      - source_labels: []
        regex: .*
        target_label: __address__
        replacement: remote.site:9115
```